charts/cray-spire/Chart.yaml

```diff
@@ -23,13 +23,16 @@
 ---
 apiVersion: v2
 name: cray-spire
-version: 1.1.3
+version: 1.2.0
 description: A Helm chart for spire
 home: https://github.com/Cray-HPE/cray-spire
 dependencies:
   - name: cray-service
```

Comment:
Should cray-service be removed? All it was used for is postgresql. This can be removed from values.yaml as well.

Reply:
I've removed the dependency on cray-service.